Source/EventFlow/EventStores/IReliableEventPersistence.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EventFlow.Aggregates;
+using EventFlow.Core;
+
+namespace EventFlow.EventStores
+{
+    public interface IReliableEventPersistence: IEventPersistence
```

Comment:
Might as well expand the `IEventPersistence` interface with the new methods since its the event store itself that states if its able to handle the requests. Then throw `NotImplementedException` in the event stores that doesn't have the capability.

This would require moving it to 1.0 though...

Reply:
agreed,
I split it so it can be compiled with current stores